cli/index.js

```
@@ -0,0 +1,82 @@
+        });
+    };
+    const help = Action.help;
+    cmd.description(help.description).action(actionHandler);
+    help.summary && cmd.summary(help.summary);
+    help.options && help.options.forEach(([ arg, desc, choices]) => {
+        if (choices) {
+            cmd.addOption(new Option(arg, desc).choices(choices));
+        } else {
+            cmd.option(arg, desc);
+        }
+    });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)